h2o-py/tests/testdir_generic_model/pyunit_generic_model_mojo_xgboost.py

```diff
@@ -0,0 +1,76 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+import tempfile
+import os
+
+from h2o.estimators import H2OXGBoostEstimator, H2OGenericEstimator
+from tests import pyunit_utils
+from tests.testdir_generic_model import Capturing, compare_output
+
+
+def test(x, y, output_test, strip_part, algo_name, generic_algo_name):
+    airlines = h2o.import_file(path=pyunit_utils.locate("smalldata/testng/airlines_train.csv"))
+    xgb = H2OXGBoostEstimator(ntrees=1, nfolds=3)
+    xgb.train(x=x, y=y, training_frame=airlines, validation_frame=airlines)
+    print(xgb)
+    with Capturing() as original_output:
+        xgb.show()
+    
+    original_model_filename = tempfile.mkdtemp()
+    original_model_filename = xgb.download_mojo(original_model_filename)
```

Comment:
Just cosmetics, but the old formula is confusing to me. :) But nothing important!

```suggestion
    original_model_filename = xgb.download_mojo(tempfile.mkdtemp())
```